vhdbuilder/packer/pre-install-dependencies.sh

```diff
@@ -20,7 +20,6 @@ CPU_ARCH=$(getCPUArch)  #amd64 or arm64
 VHD_LOGS_FILEPATH=/opt/azure/vhd-install.complete
 COMPONENTS_FILEPATH=/opt/azure/components.json
 PERFORMANCE_DATA_FILE=/opt/azure/vhd-build-performance-data.json
-MANIFEST_FILEPATH=/opt/azure/manifest.json
 #this is used by post build test to check whether the compoenents do indeed exist
 cat components.json > ${COMPONENTS_FILEPATH}
 cat manifest.json > ${MANIFEST_FILEPATH}
```

Comment:
im assuming this line shouldn't be here anymore

Reply:
True, removed it.